composer.json

```diff
@@ -28,7 +28,7 @@
     },
     "require": {
         "php": ">=7.0",
-        "nikic/php-parser": "^3.0.4",
+        "nikic/php-parser": "^3.0.5",
```

Comment:
Its best practice for libs to not have it. But its also best practice to define the dependencies as low as possible (since composer install will choose the latest possible combinations anyway)